src/core/qgslegendrenderer.h

```diff
@@ -119,17 +119,17 @@ class CORE_EXPORT QgsLegendRenderer
 #ifndef SIP_RUN
 
     /**
-     * A legend Nucleon is either a group title, a layer title or a layer child item.
+     * A legend component is either a group title, a layer title or a layer child item.
```

Comment:
Nice terminology renaming here, makes the code a bit easier to read.

Reply:
Yeah, that atom/nucleon thing always trips me up. Plus, the analogy doesn't actually match, which makes it even more confusing..!

Still need to cleanup the cpp side to reflect this too